### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by alivaditis**
### What is the feature?
Featured recipes are shown on the landing page by tag

### How you implemented the solution?
2 new functions
updateFeaturedRecipes
renderFeaturedRecipes

### Any changes in the UI (Screenshots)?

Screen Shot 2023-05-21 at 5 26 58 PM

### How to test the feature (A test scenario or any new setup)?
checkout the branch run npm start and test for functionality in the browser at the local host

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -131,6 +135,20 @@ const updateRecipeOfTheDay = () => {
...
 
+const updateFeaturedRecipes = () => {
+  featuredRecipes = []
+  const tag = getRandomItem(getAllTags(recipeData))
+  const taggedRecipes = filterRecipes(recipeData, tag)
+  featuredRecipes = taggedRecipes.slice(0,4)
+  if(featuredRecipes.length < 4) {
+    updateFeaturedRecipes()
+  } else {
+    const capitalTag = tag.split(' ').map(substring => substring[0].toUpperCase() + substring.slice(1)).join(' ')
```
Comment (paulina-isabel) [inline src/scripts.js:146]: Wow!

**Review by paulina-isabel — APPROVED** [review]
This looks so good!

**Review by jalbe0076 — APPROVED** [review]
On smaller screens, we only see half of the recipe. We will need to fix this for the second part of the project.
image

**Comment by alivaditis** [comment]
Yeah the height of the two sections is something we should mess around with, though here I wasn't trying to fit it all on one screen necessarily, anticipating scrolling, I tried out some different things like the recipe of the day taking up all of the screen initially and then scrolling down to see the featured recipes, but we should experiment with different things.

**Review by CaliHam — APPROVED** [review]
Nice!! It looks so good! Great work! 😃

**Pull request merged**